app/components/wds-dialog.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+	tagName: '',
+
+	cancelText: 'Cancel',
+	okText: 'OK',
+
+	onCancel() {},
+	onOK() {},
+
+	actions: {
+		cancelBubble() {}
```

Comment:
Please add a comment explaining what is this for. It's to prevent `{{action onCancel 'curtain'}}` being triggered when `wds-dialog__wrapper` or its content are clicked, right?

Reply:
yes